.github/CODEOWNERS

```diff
@@ -39,14 +39,16 @@ package.json              @DataDog/corpweb @DataDog/documentation
 /layouts/shortcodes/integration-assets-reference.md                        @Datadog/marketplace-product-management @DataDog/documentation
 /layouts/shortcodes/integration_categories.md                              @Datadog/marketplace-product-management @DataDog/documentation
 
-# Websites
-.github/workflows                                       @DataDog/corpweb
-.gitlab-ci.yml                                          @DataDog/corpweb
-Makefile                                                @DataDog/corpweb
-translate.yaml                                          @DataDog/corpweb
-.translate                                              @DataDog/corpweb
-docker-compose-docs.yml                                 @DataDog/corpweb
-.htmltest.yml                                           @DataDog/corpweb
+# Websites Platform (Devops)
+.github/workflows                                       @DataDog/WebOps-Platform
+.gitlab-ci.yml                                          @DataDog/WebOps-Platform
+Makefile                                                @DataDog/WebOps-Platform
+translate.yaml                                          @DataDog/WebOps-Platform
+.translate                                              @DataDog/WebOps-Platform
+docker-compose-docs.yml                                 @DataDog/WebOps-Platform
+.htmltest.yml                                           @DataDog/WebOps-Platform
+datadog-ci.preview.json                                 @DataDog/WebOps-Platform
+general.preview.synthetics.json                         @DataDog/WebOps-Platform
```

Comment:
We were already required viewers on most of these files, now we're just getting more specific with the scope of who needs to review these files since they are devops related